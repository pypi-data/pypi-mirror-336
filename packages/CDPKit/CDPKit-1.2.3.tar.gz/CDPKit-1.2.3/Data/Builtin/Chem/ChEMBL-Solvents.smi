WATER	[OH2]
DICHLOROMETHANE	ClCCl
TRICHLOROMETHANE	ClC(Cl)Cl
ETHYL ACETATE	CCOC(=O)C
METHANOL	CO
PROPAN-2-OL	CC(C)O
ACETONE	CC(=O)C
DMSO	CS(=O)C
ETHANOL	CCO
