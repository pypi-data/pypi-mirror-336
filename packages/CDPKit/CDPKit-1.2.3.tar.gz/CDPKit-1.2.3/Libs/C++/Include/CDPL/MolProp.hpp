/* 
 * MolProp.hpp 
 *
 * This file is part of the Chemical Data Processing Toolkit
 *
 * Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with this library; see the file COPYING. If not, write to
 * the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 * Boston, MA 02111-1307, USA.
 */

/**
 * \file
 * \brief A convenience header including everything that is defined in namespace CDPL::MolProp.
 */

#ifndef CDPL_MOLPROP_HPP
#define CDPL_MOLPROP_HPP

#include "CDPL/MolProp/AtomProperty.hpp"
#include "CDPL/MolProp/BondProperty.hpp"
#include "CDPL/MolProp/AtomPropertyDefault.hpp"
#include "CDPL/MolProp/MassComposition.hpp"
#include "CDPL/MolProp/ElementHistogram.hpp"
#include "CDPL/MolProp/CoordinationGeometry.hpp"
#include "CDPL/MolProp/HBondDonorAtomType.hpp"
#include "CDPL/MolProp/HBondAcceptorAtomType.hpp"
#include "CDPL/MolProp/AtomFunctions.hpp"
#include "CDPL/MolProp/BondFunctions.hpp"
#include "CDPL/MolProp/AtomContainerFunctions.hpp"
#include "CDPL/MolProp/BondContainerFunctions.hpp"
#include "CDPL/MolProp/MolecularGraphFunctions.hpp"
#include "CDPL/MolProp/TPSACalculator.hpp"
#include "CDPL/MolProp/AtomHydrophobicityCalculator.hpp"
#include "CDPL/MolProp/LogSCalculator.hpp"
#include "CDPL/MolProp/XLogPCalculator.hpp"
#include "CDPL/MolProp/PEOESigmaChargeCalculator.hpp"
#include "CDPL/MolProp/MHMOPiChargeCalculator.hpp"
#include "CDPL/MolProp/HBondDonorAtomTyper.hpp"
#include "CDPL/MolProp/HBondAcceptorAtomTyper.hpp"

#endif // CDPL_MOLPROP_HPP
