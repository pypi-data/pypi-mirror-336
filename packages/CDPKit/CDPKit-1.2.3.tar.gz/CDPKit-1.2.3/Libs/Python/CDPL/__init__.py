##
# __init__.py  
#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
##


from __future__ import absolute_import

import os
import sys
import platform

if sys.version_info[1] >= 8 and platform.system() == 'Windows':
    if 'site-packages' in __file__:
        os.add_dll_directory(os.path.dirname(__file__))
    else:
        os.add_dll_directory(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), '@CDPKIT_EXECUTABLE_INSTALL_DIR@'))

from ._cdpl import *

__version__ = VERSION_STRING
