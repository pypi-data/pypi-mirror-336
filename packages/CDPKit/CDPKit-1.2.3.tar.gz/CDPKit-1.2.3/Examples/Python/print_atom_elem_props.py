#!/bin/env python

##
# print_atom_elem_props.py 
#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# Permission to use, copy, modify, and/or distribute this software for any purpose with
# or without fee is hereby granted.
# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD
# TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN
# NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
# DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER
# IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
# CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
##


import sys

import CDPL.Chem as Chem
import CDPL.MolProp as MolProp


# outputs the corresponding properties of each atom of the provided molecular graph
def outputProperties(molgraph: Chem.MolecularGraph) -> None:
    for atom in molgraph.atoms:
        print('- Atom #%s' % str(molgraph.getAtomIndex(atom)))
        print('\tAtomic weight: %s' % str(MolProp.getAtomicWeight(atom)))
        print('\tPTE IUPAC group: %s' % str(MolProp.getIUPACGroup(atom)))
        print('\tPTE period: %s' % str(MolProp.getPeriod(atom)))
        print('\tVdW radius: %s' % str(MolProp.getVdWRadius(atom)))
        print('\tCovalent radius (bond order=1): %s' % str(MolProp.getCovalentRadius(atom, 1)))
        print('\tCovalent radius (bond order=2): %s' % str(MolProp.getCovalentRadius(atom, 2)))
        print('\tCovalent radius (bond order=3): %s' % str(MolProp.getCovalentRadius(atom, 3)))
        print('\tAllred Rochow electronegativity: %s' % str(MolProp.getAllredRochowElectronegativity(atom)))
        print('\tElement name: %s' % MolProp.getElementName(atom))
        print('\tValence electron count: %s' % str(MolProp.getElementValenceElectronCount(atom)))
        print('\tAtom type specifies chemical element: %s' % str(MolProp.isChemicalElement(atom)))
        print('\tIs main group element: %s' % str(MolProp.isMainGroupElement(atom)))
        print('\tIs metal: %s' % str(MolProp.isMetal(atom)))
        print('\tIs transition metal: %s' % str(MolProp.isTransitionMetal(atom)))
        print('\tIs non-metal: %s' % str(MolProp.isNonMetal(atom)))
        print('\tIs semi-metal: %s' % str(MolProp.isSemiMetal(atom)))
        print('\tIs halogen: %s' % str(MolProp.isHalogen(atom)))
        print('\tIs noble gas: %s' % str(MolProp.isNobleGas(atom)))

def main() -> None:
    if len(sys.argv) < 2:
        sys.exit('Usage: %s <input mol. file>' % sys.argv[0])

    # create reader for input molecules (format specified by file extension)
    reader = Chem.MoleculeReader(sys.argv[1]) 
  
    # create an instance of the default implementation of the Chem.Molecule interface
    mol = Chem.BasicMolecule()
    
    # read and process molecules one after the other until the end of input has been reached
    try:
        while reader.read(mol):
            try:
                outputProperties(mol)
            except Exception as e:
                sys.exit('Error: processing of molecule failed: ' + str(e))
                
    except Exception as e: # handle exception raised in case of severe read errors
        sys.exit('Error: reading molecule failed: ' + str(e))

    sys.exit(0)
        
if __name__ == '__main__':
    main()
