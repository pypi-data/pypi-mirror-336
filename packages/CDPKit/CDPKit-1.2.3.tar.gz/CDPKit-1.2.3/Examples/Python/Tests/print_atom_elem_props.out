- Atom #0
	Atomic weight: 18.9984032
	PTE IUPAC group: 17
	PTE period: 2
	VdW radius: 1.47
	Covalent radius (bond order=1): 0.64
	Covalent radius (bond order=2): 0.59
	Covalent radius (bond order=3): 0.53
	Allred Rochow electronegativity: 4.1
	Element name: Fluorine
	Valence electron count: 7
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: True
	Is noble gas: False
- Atom #1
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #2
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #3
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #4
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #5
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #6
	Atomic weight: 15.9994
	PTE IUPAC group: 16
	PTE period: 2
	VdW radius: 1.52
	Covalent radius (bond order=1): 0.63
	Covalent radius (bond order=2): 0.57
	Covalent radius (bond order=3): 0.53
	Allred Rochow electronegativity: 3.5
	Element name: Oxygen
	Valence electron count: 6
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #7
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #8
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #9
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #10
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #11
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #12
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #13
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #14
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #15
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #16
	Atomic weight: 14.0067
	PTE IUPAC group: 15
	PTE period: 2
	VdW radius: 1.55
	Covalent radius (bond order=1): 0.71
	Covalent radius (bond order=2): 0.6
	Covalent radius (bond order=3): 0.54
	Allred Rochow electronegativity: 3.1
	Element name: Nitrogen
	Valence electron count: 5
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #17
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #18
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #19
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #20
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #21
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #22
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #23
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #24
	Atomic weight: 14.0067
	PTE IUPAC group: 15
	PTE period: 2
	VdW radius: 1.55
	Covalent radius (bond order=1): 0.71
	Covalent radius (bond order=2): 0.6
	Covalent radius (bond order=3): 0.54
	Allred Rochow electronegativity: 3.1
	Element name: Nitrogen
	Valence electron count: 5
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #25
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #26
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #27
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #28
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #29
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #30
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #31
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #32
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #33
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #34
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #35
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #36
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #37
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #38
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #39
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #40
	Atomic weight: 12.0107
	PTE IUPAC group: 14
	PTE period: 2
	VdW radius: 1.7
	Covalent radius (bond order=1): 0.75
	Covalent radius (bond order=2): 0.67
	Covalent radius (bond order=3): 0.6
	Allred Rochow electronegativity: 2.5
	Element name: Carbon
	Valence electron count: 4
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #41
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #42
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #43
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
- Atom #44
	Atomic weight: 1.00794
	PTE IUPAC group: 1
	PTE period: 1
	VdW radius: 1.2
	Covalent radius (bond order=1): 0.32
	Covalent radius (bond order=2): 0.0
	Covalent radius (bond order=3): 0.0
	Allred Rochow electronegativity: 2.2
	Element name: Hydrogen
	Valence electron count: 1
	Atom type specifies chemical element: True
	Is main group element: True
	Is metal: False
	Is transition metal: False
	Is non-metal: True
	Is semi-metal: False
	Is halogen: False
	Is noble gas: False
